adsws/api/discoverer/views.py

```diff
@@ -18,9 +18,15 @@ def __init__(self, endpoint, service_uri, deploy_path, route):
         self.cs = None
         try:
             self.default_request_timeout = current_app.config.get("DEFAULT_REQUEST_TIMEOUT", 60)
+            self.pool_connections = current_app.config.get("REQUESTS_POOL_CONNECTIONS", 20)
+            self.pool_maxsize = current_app.config.get("REQUESTS_POOL_MAXSIZE", 1000)
+            self.max_retries = current_app.config.get("REQUESTS_MAX_RETRIES", 1)
```

Comment:
`REQUESTS_MAX_RETRIES` does not exist in the config file but `REQUESTS_POOL_RETRIES` does, could you pick one and correct the other?

Reply:
Thanks for spotting this. Fixed!